includes/ui/settings.php

```diff
@@ -93,19 +98,31 @@ class="section section-<?php echo esc_attr( $tab->slug ); ?> <?php echo $tab->de
                     </p>
 
                     <ul>
-                        <?php if ( ! $isPhp7 ) : ?>
+                        <?php if ( ! $is_php7 ) : ?>
                             <li>
-                                <?php printf( esc_html__( 'The current version of PHP (%s) is too old. PHP 7.0 or newer is required.', 'redis-cache' ), phpversion() ); ?>
+                                <?php
+                                    printf(
+                                        // translators: %s = PHP Version.
+                                        esc_html__( 'The current version of PHP (%s) is too old. PHP 7.0 or newer is required.', 'redis-cache' ),
+                                        esc_html( phpversion() )
+                                    );
+                                ?>
                             </li>
                         <?php endif; ?>
 
-                        <?php if ( ! $phpRedisInstalled ) : ?>
+                        <?php if ( ! $phpredis_installed ) : ?>
                             <li>
-                                <?php printf( esc_html__( 'The PhpRedis extension is not installed.', 'redis-cache' ), phpversion() ); ?>
+                                <?php printf( esc_html__( 'The PhpRedis extension is not installed.', 'redis-cache' ), esc_html( phpversion() ) ); ?>
```

Comment:
We can remove the `printf()` eh?

Reply:
Absolutely right!
If you do too many of these changes you are going to get blind eventually :wink: